Assets/Character/AaronTestAssets/Player.cs

```diff
@@ -0,0 +1,60 @@
+﻿using UnityEngine;
+using System.Collections;
+
+
+
+public class Player : MonoBehaviour {
+
+
+    public float speed = 2f;
+    private Animator animator;
+    private Rigidbody2D rigidbody;
+	// Use this for initialization
+	void Start () {
+
+        animator = GetComponent<Animator>();
+        rigidbody = GetComponent<Rigidbody2D>();
+       // if (animator.)
+	
+	}
+	
+	// Update is called once per frame
+	void Update () {
+        int horizontal = 0;
+        
+
+        horizontal = (int)(Input.GetAxisRaw("Horizontal"));
```

Comment:
GetAxisRaw will return -1, 0, or 1; GetAxis has "smoothing filtering" which right now just adds a large delay to pressing left and right arrow keys.